web/src/hooks/useInfiniteScroll.ts

```diff
@@ -6,10 +6,20 @@ type TUseInfiniteScrollParams<P> = P & {
   take?: number;
 };
 
+export interface InfiniteScrollModel<T> {
+  isLoading: boolean;
+  localPage: number;
+  loadMore: () => void;
+  hasMore: boolean;
+  isFetching: boolean;
+  refresh: () => void;
+  list: [T[], React.Dispatch<React.SetStateAction<T[]>>][0];
```

Comment:
is this the same as:
```ts
list: T[];
```

Reply:
right! I have auto-generated this with Intellij.
@schoren is working on a couple of improvements for the backend
then I can change this

Reply:
@cescoferraro ready